File: Logica JS/log9.js
function multNumber(){
    let multiplicando = 0;
    let multiplicador = 0;
    let soma = 0;
    let cont = 0;
    let outputElement = document.getElementById('output');

    multiplicando = parseInt(prompt("Informe o multiplicando: "));
    multiplicador = parseInt(prompt("Informe o multiplicador: "));

    for(let i=0; i<multiplicador; i++){
        soma += multiplicando;
    }

    outputElement.innerHTML += "Resultado: "+soma+"";
}

multNumber()

File: Logica C/log9.c
#include <stdio.h>

int main() {
    int multiplicando = 0, multiplicador = 0;
    int soma = 0, cont = 0;

    printf("Informe o multiplicando: \n");
    scanf("%d", &multiplicando);

    printf("Informe o multiplicador: ");
    scanf("%d", &multiplicador);

    for (int i = 0; i < multiplicador; i++) {
        soma+= multiplicando; //soma = soma + multiplicando
      }

    printf("Resultado: %d\n", soma);

    return 0;
}